### ISSUE TITLE
Certificate based ACL

### ISSUE THREAD

**Issue by nerdy-sam** [umurmur/umurmur#47]
What do you think about a login ACL based off of CA signed certs?

**Comment by rindeal** [comment]
This is a duplicate of #35 I think

**Comment by fmorgner** [comment]
I don't believe this to be duplicate, since @mitzip wants to be able to allow only clients with a cert singed by some specific CA to be able to join the server in the first place, while #35 targets controlling access to certain channels based on the certificate itself.

**Comment by klausenbusk** [comment]
+1

**Comment by rindeal** [comment]
@fmorgner yeah, i see the difference now. Well, maybe there should be created a new issue, covering ACL as a top level feature, which would then centralize all suggestions for ACL rules like: certificate based, CA based, IP based, IP range based, ...
